docs/src/main/tut/auth.md

```diff
@@ -42,11 +43,11 @@ redirect to a login page, or a popup requesting login data. With the upcoming of
 ### With Kleisli
 
 To allow for failure, the `authUser` function has to be adjusted to a `Request
-=> Task[String \/ User]`. So we'll need to handle that possibility. For advanced
+=> Task[String Either User]`. So we'll need to handle that possibility. For advanced
```

Comment:
At this point, I've just done a simple search and replace for \/. I can go through and replace these with the Either[String, User] style if that is preferred.